README.md

```diff
@@ -1,6 +1,12 @@
 # Hello Microverse
```

Comment:
- [ ] Linter should be set up correctly for this repository, but they are not running, kindly check this out

Reply:
Linter now is running ok.